public/index.html

```diff
@@ -9,6 +9,10 @@
       content="Web site created using create-react-app"
     />
     <title>hello bylaws</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,669;0,700;0,800;0,900;1,400;1,500;1,600;1,669;1,700;1,800;1,900&display=swap"
```

Comment:
You are loading a lot of different font-weights. Do you really need all of them? If not you should clean up later.

Reply:
Good point - I just linked all fonts in advance. I will clean it up!

Reply:
Done